### PR TITLE
Use SVG icons for agent context menu and executor status

### PR DESCRIPTION
**Pull request by zbynek**
### Proposed changelog entries

* Use SVG icons for agent context menu and executor status.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [n/a] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/lib/hudson/executors.jelly
Comment (timja) [inline core/src/main/resources/lib/hudson/executors.jelly:41]: looks like some left over debug?
```suggestion
          <j:when test="${c.offlineCause!=null and !c.connecting}">
```

**Review by NotMyFault — COMMENTED** [review]
Does that actually need to be merged independently into master? There are a plenty of occasions where GIFs and PNGs are used still, but applicable SVGs are present.
https://github.com/jenkinsci/jenkins/pull/5778 addresses many, if not all, of these locations.
Though that specific location in `executors.jelly` seems to be missed out in named PR.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
> Does that actually need to be merged independently into master

Generally better to incrementally merge, so things don't get held up forever.

**Review by NotMyFault — APPROVED** [review]
Good point timja, wish my other's PR list wouldn't take that long to unblock the PR 👀 

But this one looks fine based on my testing.

**Comment by zbynek** [comment]
I checked #5778 and it switches the computer icons to SVGs, but does not change the folder from `24x24` to `svgs`, resulting in some missing images.
image

I'd suggest to first merge this, then update #5778.

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by Bananeweizen** [comment]
For me all the small size icons are off center. The big icons on the status page are okay: 
image

**Pull request merged**
